modules/core/src/test/java/org/apache/ignite/testsuites/PartitionReconciliationTestSuite.java

```diff
@@ -74,6 +76,8 @@
         GridTestUtils.addTestIfNeeded(suite, ReconciliationResourceLimitedJobTest.class, ignoredTests);
         GridTestUtils.addTestIfNeeded(suite, PartitionReconciliationFastCheckTest.class, ignoredTests);
         GridTestUtils.addTestIfNeeded(suite, PartitionReconciliationCompactCollectorTest.class, ignoredTests);
+        GridTestUtils.addTestIfNeeded(suite, PartitionReconciliationAtomicLongDataStructureTest.class, ignoredTests);
+        GridTestUtils.addTestIfNeeded(suite, PartitionReconciliationSetDataStructureTest.class, ignoredTests);
```

Comment:
Please consider adding sys caches to partition reconciliation stress tests.

Reply:
Added, but --repair seems to be failing on AtomicLong, investigating

Reply:
I'll create a separate ticket for that. For now, it seems to be working in case of a non-stress scenario.